### PR TITLE
{bio}[foss/2019b,foss/2020a,foss/2020b] SHAPEIT4 v4.2.0

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0785b0b25ce878cdfaea850f2c3cd9ae for a full test report.

File: easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4_4.2.0_use_EB_settings.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
+--- shapeit4-4.2.0/makefile	2021-01-21 08:05:26.000000000 +0000
```
Comment (branfosj) [inline easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4_4.2.0_use_EB_settings.patch:1]: Please add a description and author information to the patch.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4_4.2.0_use_EB_settings.patch (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4_4.2.0_use_EB_settings.patch:4]: Done. It's essentially just a re-jigging of the previous patch.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/94dfd207a300aa08b020cca062b0702f for a full test report.

**Pull request merged**